conseil-api/src/main/scala/tech/cryptonomic/conseil/api/util/RetryStrategy.scala

```diff
@@ -0,0 +1,17 @@
+package tech.cryptonomic.conseil.api.util
+
+import tech.cryptonomic.conseil.api.ConseilApi.NoNetworkEnabledError
+
+object RetryStrategy {
+
+  /** Method which defines custom strategy, when retry mechanism should not continue
+    *
+    * Currently retry stops when:
+    * - there is no network enabled in the configuration
+    * */
+  val retryGiveUpStrategy: Throwable => Boolean = {
+    case _: NoNetworkEnabledError => true
```

Comment:
Is the error message properly shown on the console when this happens? Is there a better way to log it, so that we make sure it will also show on logstash?

Reply:
<img width="1362" alt="Zrzut ekranu 2020-07-22 11 47 04" src="https://user-images.githubusercontent.com/15884184/88162096-2a098a00-cc11-11ea-8250-41994505ee45.png">

This is how it looks. Like regular exception I guess, unless you want me to put it in single line instead of indentation?

Reply:
It's good